backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/JwtService.kt

```diff
@@ -13,7 +14,7 @@ object JwtService {
     private val secret by lazy {
         System.getenv("JWT_SECRET")
             ?: readJwtSecretFile(System.getenv("JWT_SECRET_FILE"))
-            ?: throw NoSuchElementException("Environment variable JWT_SECRET not set")
+            ?: throw MissingAuthorizationException()
```

Comment:
Hmm. I tried to keep this behavior the same as before. An `UnauthorizedException` would lead to 401, while the `NoSuchElementException` would be forwarded to the frontend. But maybe you're right. I'll change it to `UnauthorizedException`